src/index.js

```diff
@@ -1 +1,12 @@
+/* eslint-disable no-console */
 'use strict';
+
+require('dotenv/config');
+
+const { createServer } = require('./createServer');
+
+const PORT = process.env.PORT || 3005;
+
+createServer().listen(PORT, () => {
+  console.log('Server is running on localhost:5700');
```

Comment:
The console log message incorrectly states that the server is running on 'localhost:5700'. It should reflect the actual port the server is listening on, which is stored in the 'PORT' variable.